doc/dev/release-checklists.rst

```diff
@@ -18,15 +18,13 @@ Versions and tags
 - [x] Update CMakeLists.txt VERSION (right at the top to X.0.0)
 - [x] Update src/librbd/CMakeLists.txt VERSION (librbd target at the bottom to 1.X.0)
 - [x] Update src/ceph_release with the new release name, number, and type ('dev')
-- [ ] Initial tag vX.0.0 (so that we can distinguish from (and sort
-      after) the backported (X-1).2.Z versions.
-
-### Notes on tagging
-* Tags must be annonated as CMake determines `CEPH_GIT_NICE_VER` by
-calling `git describe --always`.
-* vX.0.0 are special ones in the sense they are pushed manually (unlike v.X.2.n
-which are handled by Jenkins).
-* vX.0.0 should point to a commit before the first one in a kickoff branch.
+- [x] Initial tag vX.0.0 (so that we can distinguish from (and sort after) the backported (X-1).2.Z versions.
+
+.. note::
+
+    - Tags must be annonated as CMake determines ``CEPH_GIT_NICE_VER`` by calling ``git describe --always``.
+    - vX.0.0 are special ones in the sense they are pushed manually (unlike v.X.2.n which are handled by Jenkins).
+    - vX.0.0 should point to a commit before the first one in a kickoff branch.
```

Comment:
@rzarzynski is this actually necessary? Why not point to the merge commit of the kickoff?

Reply:
Noting that the recently created v20.0.0 is pointing to 048fc68c517f50b9978457f478ca4638f01caa09 which is the merge commit that preceded the merge of the kickoff (i.e. the first parent of the merge commit of the kickoff).  At the same time one might interpret "a commit before the first one in a kickoff branch" as the base of the kickoff branch which would be 7f414b23c2b754f5bfa89a7ae62c6422c55b9941.  This bullet really needs to be clarified.

Reply:
> @rzarzynski is this actually necessary? Why not point to the merge commit of the kickoff?

The merge commit has two parents: a commit in `main` and the very last commit in corresponding kickoff branch. Tagging it would mean initial kickoff's commits are not considered as the release's ones leading to wrong version string generated by `git describe`. I guess the consequence is problems with local testing.

I'm not sure taking merge's _left_ parent changes much in this aspect.

IIRC the quincy and pacific tags are even deeper in their corresponding branches suggesting there was a rebase wthout updating the tag.

Reply:
@rzarzynski So the expectation is for the tag to be on the base of the kickoff branch, meaning 7f414b23c2b754f5bfa89a7ae62c6422c55b9941 for tentacle?

Reply:
I think so.

Reply:
@rzarzynski Does v20.0.0 tag need to be adjusted in your opinion?